docs/quick-visual-tutorials/visual-ibac-istio-authorization-policies.mdx

```diff
@@ -8,15 +8,325 @@ import CodeBlock from "@theme/CodeBlock";
+details when creating the network map. That way we not only have better visibility of the calling patterns,
+e.g. in the access graph, but we can also use that information to automatically generate fine-grained intents and
+enforce them with Istio authorization policies.
 :::
+
+## Deploy demo to simulate traffic
+<details>
+<summary>Create a namespace for our demo application and label it for Istio injection</summary>
+
+```bash
+kubectl create namespace otterize-visual-tutorial-istio
+kubectl label namespace otterize-visual-tutorial-istio istio-injection=enabled
+```
+</details>
+
+<details>
+<summary>Deploy the demo set of services</summary>
+
+```
+kubectl apply -n otterize-visual-tutorial-istio -f https://docs.otterize.com/code-examples/ibac-for-istio/demo-app.yaml
+```
+</details>
+
+## Seeing the access graph
+
+In the Otterize Cloud UI, your [cluster](https://app.otterize.com/clusters) should now show all 3 Otterize OSS operators &mdash; the intents operator, network mapper, and credentials operator &mdash; as connected, with a green status.
+
+And when you go back to the [access graph](https://app.otterize.com/access-graph) (and select your cluster from the dropdown, if needed), you should see the following map for the demo running in your cluster:
+
+![Access graph](/img/quick-tutorials/istio-visual-tutorial/phase-0.png)
+
+Each service is shown as a node in the access graph, while the dashed lines (edges) connecting the services show access between them, as detected by the network mapper.
+We need to adjust the view to Istio authorization policies, so enable the Istio "Use in access graph" button and disable the NetworkPolicies and Kafka toggle buttons.
+
+![Access graph](/img/quick-tutorials/istio-visual-tutorial/access-graph-panel.png)
+
+## Try out IBAC with shadow mode
+
+Now let's start to roll out access controls, but remain in shadow mode: no actual enforcement of controls, yet.
+
+We'll declare that the `frontend` intends to perform `GET` request to the `recommendationservice`.
+
+```yaml
+{@include: ../../static/code-examples/ibac-for-istio/phase-1.yaml}
+```
+
+We expect this will provide secure access, allowing the intended access from the `frontend` while protecting the `recommendationservice` from unintended access.
+
+Apply this intents file with:
+```bash
+kubectl apply -n otterize-visual-tutorial-istio -f https://docs.otterize.com/code-examples/ibac-for-istio/phase-1.yaml
+```
+
+Look at the access graph again:
+
+![Access graph](/img/quick-tutorials/istio-visual-tutorial/phase-1.png)
+
+The green line from `frontend` to `recommendationservice`, representing the discovered intent from the network mapper, no longer dashed, but rather **solid**.
+
+Click on that `frontend` &rarr; `recommendationservice` line:
+<img src="/img/quick-tutorials/istio-visual-tutorial/frontend-recommendation-applied.png" alt="Discovered intents" width="600"/>
+
```

Comment:
The screenshot currently shows a missing declared intent; please replace with an image showing the appropriate UI.